tests/cypress/support/commands.js

```diff
@@ -490,3 +490,19 @@ Cypress.Commands.add('writeFilterValue', (clear, filterValue) => {
             cy.get('.ant-select-selection__choice__content').should('have.text', filterValue);
         });
 });
+
+Cypress.Commands.add('checkFrameNum', (frameNum) => {
+    cy.get('.cvat-player-frame-selector').within(() => {
+        cy.get('input[role="spinbutton"]').should('have.value', frameNum);
+    });
+});
+
+Cypress.Commands.add('goToNextDefaultFrame', (expectedFrameNum) => {
```

Comment:
why "default"? Just ``goToNextFrame``. 
Haven't we had such the command till the present time?

Reply:
Function renamed. This function did not use in global file (commands.js).